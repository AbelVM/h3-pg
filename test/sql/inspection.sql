\set invalid '\'0\''
\set hexagon '\'880326b88dfffff\''
\set pentagon '\'831c00fffffffff\''

SELECT h3_h3_is_pentagon(:hexagon), h3_h3_is_pentagon(:pentagon);

SELECT h3_h3_is_valid(:hexagon), h3_h3_is_valid(:pentagon), h3_h3_is_valid(:invalid);

SELECT h3_h3_get_resolution(:hexagon), h3_h3_get_resolution(:pentagon);

SELECT h3_h3_get_base_cell(:hexagon), h3_h3_get_base_cell(h3_h3_to_parent(:hexagon));
SELECT h3_h3_get_base_cell(:pentagon), h3_h3_get_base_cell(h3_h3_to_parent(:pentagon));

SELECT h3_h3_is_res_class_iii(:hexagon), h3_h3_is_res_class_iii(h3_h3_to_parent(:hexagon));
SELECT h3_h3_is_res_class_iii(:pentagon), h3_h3_is_res_class_iii(h3_h3_to_parent(:pentagon));