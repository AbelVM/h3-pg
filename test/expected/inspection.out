\set invalid '\'0\''
\set hexagon '\'880326b88dfffff\''
\set pentagon '\'831c00fffffffff\''
SELECT h3_h3_is_pentagon(:hexagon), h3_h3_is_pentagon(:pentagon);
 h3_h3_is_pentagon | h3_h3_is_pentagon 
-------------------+-------------------
 f                 | t
(1 row)

SELECT h3_h3_is_valid(:hexagon), h3_h3_is_valid(:pentagon), h3_h3_is_valid(:invalid);
 h3_h3_is_valid | h3_h3_is_valid | h3_h3_is_valid 
----------------+----------------+----------------
 t              | t              | f
(1 row)

SELECT h3_h3_get_resolution(:hexagon), h3_h3_get_resolution(:pentagon);
 h3_h3_get_resolution | h3_h3_get_resolution 
----------------------+----------------------
                    8 |                    3
(1 row)

SELECT h3_h3_get_base_cell(:hexagon), h3_h3_get_base_cell(h3_h3_to_parent(:hexagon));
 h3_h3_get_base_cell | h3_h3_get_base_cell 
---------------------+---------------------
                   1 |                   1
(1 row)

SELECT h3_h3_get_base_cell(:pentagon), h3_h3_get_base_cell(h3_h3_to_parent(:pentagon));
 h3_h3_get_base_cell | h3_h3_get_base_cell 
---------------------+---------------------
                  14 |                  14
(1 row)

SELECT h3_h3_is_res_class_iii(:hexagon), h3_h3_is_res_class_iii(h3_h3_to_parent(:hexagon));
 h3_h3_is_res_class_iii | h3_h3_is_res_class_iii 
------------------------+------------------------
 f                      | t
(1 row)

SELECT h3_h3_is_res_class_iii(:pentagon), h3_h3_is_res_class_iii(h3_h3_to_parent(:pentagon));
 h3_h3_is_res_class_iii | h3_h3_is_res_class_iii 
------------------------+------------------------
 t                      | f
(1 row)

